chroma_core/lib/stratagem.py

```diff
@@ -16,6 +16,19 @@
     "size >= 1t": "greater_than_equal_1t",
 }
 
+labels = {
+    "less_than_1m": "<\\\ 1\\\ Mib",
+    "greater_than_equal_1m_less_than_1g": ">\\\=\\\ 1\\\ Mib\\\,\\\ <\\\ 1\\\ GiB",
+    "greater_than_equal_1g": ">\\\=\\\ 1\\\ GiB",
+    "greater_than_equal_1t": ">\\\=\\\ 1\\\ TiB",
+}
+
+distribution_weight = {
```

Comment:
```suggestion
order = {
```

`distribution_weight` makes it sound like these have different weights when the only purpose for this is to enforce ordering.

Also, since is just a k v dict of increasing order, just use a list instead.